### DISCUSSION TITLE
授權網頁404了

### DISCUSSION THREAD

**Discussion by CN-make** [xisuo67/XHS-Spider#120] [General]
你被DDOS？

**Comment by xisuo67** [comment]
能正常访问啊

**Reply by CN-make** [reply]
你是大陸的服務器？之前
![image](https://github.com/xisuo67/XHS-Spider/assets/135316273/cb527035-5b8b-4bb4-90f3-1b7f25dd0a2a)
訪問也不會受限啊，怎麼突然牆？

**Reply by xisuo67** [reply]
是的，某些原因被墙了，你搭梯子进来吧
